creme/mobile/locale/fr/LC_MESSAGES/django.po

```diff
@@ -449,24 +458,35 @@ msgid "Call in progress"
 msgstr "Appel en cours"
 
 #: templates/mobile/workflow_panel.html:14
-#, python-format
+#, fuzzy, python-format
+#| msgid ""
+#| "You left the application to call <span class='call-target'><span "
+#| "class='contact-first-name'>%(first_name)s</span> <span class='contact-"
+#| "last-name'>%(last_name)s</span></span> to <a class='phone-number' "
+#| "href='tel:%(number)s'>%(number)s</a>, you can save the result if this "
+#| "call."
 msgid ""
-"You left the application to call <span class='call-target'><span "
-"class='contact-first-name'>%(first_name)s</span> <span class='contact-last-"
-"name'>%(last_name)s</span></span> to <a class='phone-number' href='tel:"
-"%(number)s'>%(number)s</a>, you can save the result if this call."
+"You left the application to call <span class=\"call-target\"><span class="
+"\"contact-first-name\">%(first_name)s</span> <span class=\"contact-last-name"
+"\">%(last_name)s</span></span> to <a class=\"phone-number\" href=\"tel:"
+"%(number)s\">%(number)s</a>, you can save the result if this call."
 msgstr ""
 "Vous avez quitté l'application pour appeler <span class='call-target'><span "
 "class='contact-first-name'>%(first_name)s</span> <span class='contact-last-"
 "name'>%(last_name)s</span></span> au <a class='phone-number' href='tel:"
 "%(number)s'>%(number)s</a>, vous pouvez sauvegarder le résultat de cet appel."
 
 #: templates/mobile/workflow_panel.html:18
-#, python-format
+#, fuzzy, python-format
+#| msgid ""
+#| "You left the application to call <span class='call-target'><span "
+#| "class='orga-name'>%(name)s</span></span> to <a class='phone-number' "
+#| "href='tel:%(number)s'>%(number)s</a>, you can save the result if this "
+#| "call."
```

Comment:
Junk

Reply:
I call that a gettext-collect surprise :smile: